spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative './capybara_drivers'
+
+module FeatureHelpers
+  def sign_in(user: nil)
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:psu] = mock_auth_hash(user) if user.present?
+  end
+
+  private
+
+    def mock_auth_hash(user)
+      build :psu_oauth_response, access_id: user.access_id
+    end
+end
+
+RSpec.configure do |config|
+  Capybara.javascript_driver = if ci_build?
+                                 :selenium_remote
+                               else
+                                 :selenium_chrome_headless
+                                 # :selenium_chrome # For debugging in dev
+                               end
```

Comment:
@rschenk I think this block could just get moved to to the CapybaraDrivers module, and then you wouldn't need the `require_relative` anymore.

Reply:
We don't actually need the `require_relative` at all, I just put it there as a little hint to where `:selenium_remote` was coming from

I guess I figured that the capybara_drivers file would only contain driver definitions, whereas the feature_helpers file would be for configuring how you wanted the feature specs to work?

Reply:
@rschenk I get that. The ontologist in me is saying put it in CapybaraDrivers because it _says_ Capybara 😄 I can go either way...